### PR TITLE
NOS97 - Change relay and contact list

### PR DESCRIPTION
**Pull request by cjorgensen**
- Removes Relay reference from Follow
- Adds Relay(s) reference to Author
- Updates RelayView and RelayService to use CurrentUser.author.relays instead of all Relays

### CODE REVIEW

**Comment by jcswift23** [comment]
I tested this out, but it wasn't updating new posts on my home feed, will test it more to see what's going on.

**Review by jcswift23 — APPROVED** [review]
Things seem to work fine on a fresh install for me.

File: Nos/Views/AppView.swift [hunk 1]
```diff
@@ -83,7 +83,7 @@ struct AppView: View {
                                 .tabItem { Label(Localized.notifications.string, systemImage: "bell") }
                                 .tag(Destination.notifications)
 
-                            RelayView()
```
Comment (mplorentz) [inline Nos/Views/AppView.swift:86]: I think if the user logs out or changes their key in Settings the author the RelayView has a reference to won't change. I bet this is a bug with HomeFeedView too. I'll open a separate ticket for this.

Comment (mplorentz) [reply Nos/Views/AppView.swift:86]: Created #128

File: Nos/Service/CurrentUser.swift [hunk 2]
```diff
@@ -38,7 +38,9 @@ enum CurrentUser {
     }
     
     static var author: Author {
-        try! Author.findOrCreate(by: publicKey!, context: context!)
+        let persistenceController = PersistenceController.shared
+        context = persistenceController.container.viewContext
+        return try! Author.findOrCreate(by: publicKey!, context: context!)
```
Comment (mplorentz) [inline Nos/Service/CurrentUser.swift:43]: I tried removing my key from Settings and relaunching the app and it crashes on this line. Maybe we should make this optional?

**Review by mplorentz — COMMENTED** [review]
I found a couple issues, but it's looking good overall.

File: Nos/Service/CurrentUser.swift (hunk 2 above)
Comment (cjorgensen) [reply Nos/Service/CurrentUser.swift:43]: Ah, good catch. Yes, I'l fix that.

**Pull request merged**